### ISSUE TITLE
Test fails

### ISSUE THREAD

**Issue by abhijithch** [JuliaLang/Roots.jl#41]
`Pkg.test("Roots")` fails on `Julia v0.5`, the error is 
```
Error During Test
  Test threw an exception of type ErrorException
  Expression: (fzeros((x->begin 
                x - 1
            end)))[1] == 1.0
  factor(1,) has been moved to the package Primes.jl.
  Run Pkg.add("Primes") to install Primes on Julia v0.5-
   in factor(::BigInt) at ./deprecated.jl:210

```

**Comment by abhijithch** [comment]
Will try to fix this and send a PR.

**Comment by jverzani** [comment]
Thanks for reporting. This is closed now with 2a61b9f2ae4cbabecf15a0190996b160d72de185 . However, there seems to be performance regressions with v0.5 (#42)
